app/controllers/editorial/submissions_controller.rb

```diff
@@ -9,6 +9,12 @@ class Editorial::SubmissionsController < Editorial::SectionsController
     before_action :find_submission, only: [:show, :update]
     decorates_assigned :submission
 
+    def show
+      super
+      # FIXME: this works around the node Fabricator not setting the type in specs
```

Comment:
Yeah it's not a fun job but tests that create instances of the base class aren't testing things production does in Real Life™ 😜